server/server.go

```diff
@@ -37,26 +39,20 @@ type Server struct {
 
 	ActivityManager *ActivityManager
 
-	LicenseService enterprise.LicenseService
-
-	e *echo.Echo
-
-	store         *store.Store
-	l             *zap.Logger
-	lvl           *zap.AtomicLevel
-	version       string
-	mode          common.ReleaseMode
-	host          string
-	port          int
-	frontendHost  string
-	frontendPort  int
-	datastorePort int
-	startedTs     int64
-	secret        string
-	readonly      bool
-	demo          bool
-	dataDir       string
-	subscription  *enterprise.Subscription
+	LicenseService enterpriseAPI.LicenseService
+
+	e            *echo.Echo
+	profile      *Profile
+	metaDB       *metadb.MetadataDB
+	db           *store.DB
+	store        *store.Store
+	l            *zap.Logger
+	lvl          *zap.AtomicLevel
+	startedTs    int64
+	secret       string
+	subscription *enterpriseAPI.Subscription
```

Comment:
nit: since we are here, could you please put subscription *enterpriseAPI.Subscription below LicenseService enterpriseAPI.LicenseService

Reply:
SG.